e2e/test.yaml

```diff
@@ -9,3 +9,4 @@ spec:
     configMap:
       name: "crocodile-stress-test"
       file: "test.js"
+  arguments: -v
```

Comment:
Could you please remove this change? Often it's the "first example" file that new users try and I don't think it should contain the debug output.

Reply:
Done!